pokete.py

```diff
@@ -2522,19 +2590,19 @@ def map_additions():
 
     # playmap_18
     _map = ob_maps["playmap_18"]
-    _map.lake_1 =  se.Text("""  ~~
+    _map.lake_1 = se.Text("""  ~~
  ~~~~
 ~~~~~~~
 ~~~~~~~~
 ~~~~~~~~
 ~~~~~~~~
 ~~~~~~
  ~~~~
- ~~""", esccode=Color.blue, ignore=Color.blue+" "+Color.reset,
-        ob_class=HightGrass,
-        ob_args={"pokes": ["karpi", "blub", "clampi"],
-                 "minlvl": 540, "maxlvl": 640},
-        state="float")
+ ~~""", esccode=Color.blue, ignore=Color.blue + " " + Color.reset,
+                          ob_class=HightGrass,
```

Comment:
Why is this indented.

Reply:
It is indented at the same level as the bracket in line 2593, so you can see that this code belongs to se.Text(...